src/main/java/com/hackclub/hccore/commands/StatsCommand.java

```diff
@@ -77,5 +149,42 @@ private void sendStatistics(CommandSender sender, Player player) {
         SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
         sender.sendMessage(
                 "- Registered since: " + dateFormat.format(new Date(player.getFirstPlayed())));
+
+        if (extended) {
+            sender.sendMessage("- Distance by elytra: "
+                    + toSIPrefix((double) player.getStatistic(Statistic.AVIATE_ONE_CM)) + "m");
+            sender.sendMessage("- Distance by minecart: "
+                    + toSIPrefix((double) player.getStatistic(Statistic.MINECART_ONE_CM)) + "m");
+            sender.sendMessage("- Distance walked: "
+                    + toSIPrefix((double) player.getStatistic(Statistic.WALK_ONE_CM)) + "m");
+            sender.sendMessage("- Damage taken: " + player.getStatistic(Statistic.DAMAGE_TAKEN));
+            sender.sendMessage("- Damage dealt: " + player.getStatistic(Statistic.DAMAGE_DEALT));
+            sender.sendMessage("- Times jumped: " + player.getStatistic(Statistic.JUMP));
+            sender.sendMessage("- Raids won: " + player.getStatistic(Statistic.RAID_WIN));
+            sender.sendMessage("- Diamonds gained: "
```

Comment:
It'd be cool if you made this an accurate representation of the player's diamond net worth

Reply:
its impossible to do that. tooo many ways to make it a too big number.

Reply:
its just a gimmicky statistic i guess.